### PR TITLE
Adds warning about directories blocked from being shared

### PR DESCRIPTION
**Pull request by ibeckermayer**
image

### CODE REVIEW

File: docs/pages/desktop-access/directory-sharing.mdx [hunk 1]
```diff
@@ -49,13 +49,25 @@ and click "Share Directory":
 
 ![The "Share Desktop" button](../../img/desktop-access/share.png)
 
+<Notice type="warning">
+For security reasons, the browser prevents you from sharing certain directories,
```
Comment (espadolini) [inline docs/pages/desktop-access/directory-sharing.mdx:53]:
```suggestion
For security reasons it's not possible to share certain directories,
```

File: docs/pages/desktop-access/directory-sharing.mdx [hunk 2]
```diff
@@ -49,13 +49,25 @@ and click "Share Directory":
...
+including the standard locations for commonly used user directories such as "desktop"
+and "documents", as well as critical directories more typically used by the operating
```
Comment (espadolini) [inline docs/pages/desktop-access/directory-sharing.mdx:55]:
```suggestion
including the standard locations for commonly used user directories such as "Desktop"
and "Documents", as well as critical directories more typically used by the operating
```

**Review by espadolini — APPROVED** [review]

**Review by scottpgallagher — APPROVED** [review]

File: docs/pages/desktop-access/directory-sharing.mdx [hunk 3]
```diff
@@ -49,13 +49,25 @@ and click "Share Directory":
...
 
+<Notice type="warning">
+For security reasons it's not possible to share certain directories,
+including the standard locations for commonly used user directories such as "Desktop"
+and "Documents", as well as critical directories more typically used by the operating
+system itself.
+
+For a full list of the blocked user directories, see [here](https://wicg.github.io/file-system-access/#enumdef-wellknowndirectory).
+
+Directories _within_ these blocked user directories may still be shared.
```
Comment (zmb3) [inline docs/pages/desktop-access/directory-sharing.mdx:60]:
```suggestion
Subdirectories _within_ these blocked user directories may still be shared.
```

File: docs/pages/desktop-access/directory-sharing.mdx [hunk 4]
```diff
@@ -49,13 +49,25 @@ and click "Share Directory":
 
 ![The "Share Desktop" button](../../img/desktop-access/share.png)
 
+<Notice type="warning">
+For security reasons it's not possible to share certain directories,
```
Comment (zmb3) [inline docs/pages/desktop-access/directory-sharing.mdx:53]:
```suggestion
For security reasons the web browser's filesystem access API prohibits you from sharing certain directories,
```

Let's make it clear this is a browser-imposed limitation, not a Teleport one.

**Review by zmb3 — APPROVED** [review]
Two small suggestions.

File: docs/pages/desktop-access/directory-sharing.mdx [hunk 5]
```diff
@@ -49,13 +49,25 @@ and click "Share Directory":
...
 
+<Notice type="warning">
+For security reasons the web browser's filesystem access API prohibits you from sharing certain directories,
+including the standard locations for commonly used user directories such as "Desktop"
+and "Documents", as well as critical directories more typically used by the operating
+system itself.
+
+For a full list of the blocked user directories, see [here](https://wicg.github.io/file-system-access/#enumdef-wellknowndirectory).
+
+Subdirectories _within_ these blocked user directories may still be shared.
```
Comment (zmb3) [inline docs/pages/desktop-access/directory-sharing.mdx:60]: Looks like the docs linter wants `*within*` instead.

**Pull request merged**